### PR TITLE
Add basic feature toggling support and initial two feature toggles

### LINKED ISSUE #49: Create a basic Feature Toggle / Killswitch system
This system is intended to be able to turn off a feature such as image uploaded quickly if needed.

- [x] Add a database table to track feature toggles.
- [x] Add an endpoint to read the status of a feature toggle.
- [x] Add an endpoint to update the status of a feature toggle.
- [x] Update the admin panel or page so that super users can toggle feature toggles.

### PR DESCRIPTION
**Pull request by CheesyLaZanya**
Closes #49 

Will need to run `task data:migrate` to get the new table and initial feature toggles.

Adds a basic feature toggling system we can use as super users to turn off a feature if an issue arises, or for things like merging in progress work without enabling for end users if so desired in the future.

Went with a simple approach on the front end with the assumption this is only needed for the frontend application users would use.

As implemented, feature toggles should be added with alembic to ensure they are added with the expected state for production alongside easily keeping aligned on any flags between developers.

Added two flags for now, one for the image upload, as discussed, and another one for the datasets list, as an example for one defaulting to off since that section is currently not ready for use.

### Screenshots:

**Admin page:**

image

**Home page with section removed without code changes:**

image

### CODE REVIEW

File: modules/odr_frontend/src/lib/server/pg.ts [hunk 1]
```diff
@@ -76,7 +83,31 @@ export const PG_API = {
...
+        //     return rows[0] || null;
+        // },
+        // create: async (feature_name: string, is_enabled: boolean, default_state: boolean): Promise<IFeatureToggle> => {
+        //     const { rows } = await pgClient.query(
+        //         'INSERT INTO feature_toggles (feature_name, is_enabled, default_state) VALUES ($1, $2, $3) RETURNING *',
+        //         [feature_name, is_enabled, default_state]
+        //     );
+        //     return rows[0];
+        // }
+    }
```
Comment (fearnworks) [inline modules/odr_frontend/src/lib/server/pg.ts:110]: Is this stub code for later or dead?

Comment (CheesyLaZanya) [reply modules/odr_frontend/src/lib/server/pg.ts:110]: Good catch, was leftover from some previous work, will remove.

**Comment by fearnworks** [comment]
I get this error when performing a migration with an existing db : 
sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) column "is_active" of relation "users" contains null values

We should fill the existing users with the default before applying the required constraint

**Comment by CheesyLaZanya** [comment]
> I get this error when performing a migration with an existing db : `sqlalchemy.exc.IntegrityError: (psycopg2.errors.NotNullViolation) column "is_active" of relation "users" contains null values`
> 
> We should fill the existing users with the default before applying the required constraint

Sorry, that was a poor choice on my previous is_active PR, but I have added a commit here which should smooth out that experience for anyone running migrate now: https://github.com/Open-Model-Initiative/OMI-Data-Pipeline/pull/87/commits/fe17a767441121cf5dd48003da1d737e01a6e116

Thanks!

**Review by takoyaro — APPROVED** [review]
Works as intended, LGTM!

**Pull request merged**